### PR TITLE
create default accounts when dir does not exist

### PR DESCRIPTION
**Pull request by butonic**
Fixes https://github.com/owncloud/ocis-accounts/issues/39
Signed-off-by: Jörn Friedrich Dreyer <jfd@butonic.de>

### CODE REVIEW

File: pkg/service/v0/service.go [hunk 1]
```diff
@@ -43,8 +43,92 @@ func New(cfg *config.Config) (s *Service, err error) {
...
 
-	if err = os.MkdirAll(filepath.Join(cfg.Server.AccountsDataPath, "accounts"), 0700); err != nil {
-		return nil, err
+	// check if accounts exist
+	accountsDir := filepath.Join(cfg.Server.AccountsDataPath, "accounts")
+	var fi os.FileInfo
+	if fi, err = os.Stat(accountsDir); err != nil {
+		if os.IsNotExist(err) {
+			// create accounts directory
+			if err = os.MkdirAll(accountsDir, 0700); err != nil {
```
Comment (refs) [inline pkg/service/v0/service.go:52]: `0644` to prevent spooky things

File: pkg/service/v0/service.go [hunk 2]
```diff
@@ -43,8 +43,92 @@ func New(cfg *config.Config) (s *Service, err error) {
...
+					PreferredName: "richard",
+					Mail:          "richard@example.org",
+					DisplayName:   "Richard Feynman",
+					UidNumber:     20002,
+					GidNumber:     30000,
+					PasswordProfile: &proto.PasswordProfile{
+						Password: "$6$rounds=5524$sa1u5Pmfo4cr23Vw$58bQVL/JeUlwM0RY21YKAFMvKvwKLLysGllYXox.vwKT5dHMwdzJjCxwTDMnB2o2pwexC8o/iOXyP2zrhALS40",
+					},
+				},
+				// technical users for kopao and reva
```
Comment (refs) [inline pkg/service/v0/service.go:90]: kopano*

File: pkg/service/v0/service.go (hunk 1 above)
Comment (butonic) [reply pkg/service/v0/service.go:52]: nope, this is a dir. needs executable bit ... and noone but this service should have access

**Comment by ownclouders** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pkg/service/v0/service.go  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-accounts/pullRequest?prid=5727452&bid=18738067)

**Pull request merged**